core/src/main/scala/scala/collection/parallel/RemainsIterator.scala

```diff
@@ -500,7 +501,7 @@ self =>
     }
   }
 
-  def zipParSeq[S](that: SeqSplitter[S]) = new Zipped(that)
+  def zipParSeq[S](that: SeqSplitter[S]): Zipped[S] = new Zipped(that)
```

Comment:
`def zipParSeq[S](that: SeqSplitter[S]): IterableSplitter[(T, S)] = new Zipped(that)`

Reply:
It doesn't matter one way or the other, does it?